### PR TITLE
Addition of LB to single node HANA

### PR DESCRIPTION
**Pull request by peterchoo**
PR to Master following fork PR #312 

See issue #302

PR contains:
- An update to the HANA DB TF module to add a load balancer

Prerequisites:
- Follow the [Usage Guide](https://github.com/Azure/sap-hana/blob/master/deploy/v2/USAGE.md) to ensure code configuration and authorization to Azure

Build this PR demo via:
- `util/terraform_v2.sh init`
- `util/terraform_v2.sh apply single_node_hana`

Notes:
- The health probe and LB rules can take a while to appear in the Azure portal
- There may be some problems with the ansible when testing, but that's unrelated to the TF changes in this PR (we believe)

### CODE REVIEW

**Comment by peterchoo** [comment]
Force push was rebasing to master to remove extraneous file changes.

File: deploy/v2/terraform/modules/hdb_node/main.tf [hunk 1]
```diff
@@ -73,6 +73,62 @@ resource "azurerm_network_interface" "nics-dbnodes-db" {
...
+  for_each            = local.loadbalancers
+  resource_group_name = var.resource-group[0].name
+  loadbalancer_id     = azurerm_lb.hana-lb[0].id
+  name                = "hana-${each.value.sid}-lb-hp"
+  port                = "625${each.value.instance_number}"
+  protocol            = "Tcp"
+  interval_in_seconds = 5
+  number_of_probes    = 2
+}
+# TODO:
```
Comment (rukawata) [inline deploy/v2/terraform/modules/hdb_node/main.tf]: nit: add a newline in between.

File: deploy/v2/terraform/modules/hdb_node/variables.tf [hunk 2]
```diff
@@ -39,7 +39,70 @@ locals {
...
+    "2" = [
+      "30013",
+      "30015",
+      "30040",
+      "30041",
+      "30042",
+    ]
+  }
+
+  # Hash of Load Balancers to create for
```
Comment (rukawata) [inline deploy/v2/terraform/modules/hdb_node/variables.tf]: nit: "Hash of Load Balancer to be created"?

File: util/terraform_v2.sh [hunk 3]
```diff
@@ -239,8 +239,7 @@ function check_json_template_exists()
 }
 
 
-
-# This functionn checks the auth script exists and loads it, otherwise it exits
+# This function checks the auth script exists and loads it, otherwise it exits
```
Comment (rukawata) [inline util/terraform_v2.sh]: Thanks for fixing the typo :)

**Pull request merged**